### PR TITLE
Raise maximum tag size to 96

### PR DESCRIPTION
**Pull request by noirbizarre**
Raise maximum tag length to at least support [official INSPIRE tags](http://inspire.ec.europa.eu/theme)

### CODE REVIEW

File: udata/utils.py [hunk 1]
```diff
@@ -199,9 +200,10 @@ def recursive_get(obj, key):
     return recursive_get(value, parts) if parts else value
 
 
-def unique_string(length=None):
+def unique_string(length=36):
```
Comment (davidbgk) [inline udata/utils.py]: Let's name that magic number `UUID_LENGTH`?

File: udata/utils.py [hunk 2]
```diff
@@ -199,9 +200,10 @@ def recursive_get(obj, key):
...
     '''Generate unique string'''
-    string = str(uuid4())
+    # We need a string longer than length and an UUID is 36 chars length
+    string = str(uuid4()) * int(math.ceil(length / 36.0))
     return string[:length] if length else string
```
Comment (davidbgk) [inline udata/utils.py]: `string` is the name of a standard module.

File: udata/utils.py [hunk 3]
```diff
@@ -199,9 +200,10 @@ def recursive_get(obj, key):
     return recursive_get(value, parts) if parts else value
 
 
-def unique_string(length=None):
+def unique_string(length=36):
     '''Generate unique string'''
-    string = str(uuid4())
+    # We need a string longer than length and an UUID is 36 chars length
+    string = str(uuid4()) * int(math.ceil(length / 36.0))
```
Comment (davidbgk) [inline udata/utils.py]: Given our use-case, I would just add `* 2`. Less clever but more readable :)

Comment (noirbizarre) [inline udata/utils.py]: `* 2` is not sufficient, we need at least 3.
But I spent too much time searching the cause of the test failure so lets make it work with any length.

File: udata/utils.py (hunk 2 above)
Comment (noirbizarre) [inline udata/utils.py]: Which is not imported and as it's not a builtin, there is no collision.
I changed for `uuid` but it's also a standard module and the result is not an uuid but just a string,
so I propose to keep string as it's explicit

**Pull request merged**